sources/packages/web/src/views/aest/StudentAppealRequestsApproval.vue

```diff
@@ -26,10 +26,11 @@
               color="primary"
               variant="outlined"
               class="mr-2"
+              data-cy="cancelButton"
```

Comment:
The data-cy value can be more closer to the purpose of the cancel like how submit button's data-cy is.

Reply:
Because the button's name is 'Cancel,' I used cancel in data-cy.

<img width="403" alt="image" src="https://user-images.githubusercontent.com/98920571/167528457-79effa55-20a2-4ccd-bef9-bc051c25afea.png">

Reply:
What I meant here is like cancel<something>button.  Sometimes a page can have more than one cancel actions (because of the child component it includes) and data-cy="cancel" may repeat.

Reply:
Changed text, now cancel button is more specific to Approval Request. After sync you will see changes reflected.

<img width="514" alt="image" src="https://user-images.githubusercontent.com/98920571/167704163-62a4c7ab-90eb-49b4-8e2d-1f383e13bc89.png">